### PR TITLE
version control and IDE setup

### PR DESCRIPTION
**Pull request by giselle2323**
https://github.com/Dufuna-Fem/Dufuna18/commit/41b2a2194eaa83dea6550ec77faf78a6c3599f77

### CODE REVIEW

**Comment by djubreel** [comment]
Well done, But you need to change the title of your PR to the name of the topic for the week, add a more descriptive description, remove the submissions/Dufuna18-master/README.md file that you added and also remove the versioncontrol.txt or the yakubu.txt because they both serve the same purpose. you can see a sample [here](https://github.com/Dufuna-Fem/Dufuna18/pull/7)

**Comment by giselle2323** [comment]
i have deleted them

File: Dufuna18-master/submissions/Version control/yakubu.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+I am Yakubu Aminat Jumoke.
```
Comment (djubreel) [inline Dufuna18-master/submissions/Version control/yakubu.txt]: Remove this file

File: submissions/Dufuna18-master/submissions/Yakubu Aminat Jumoke/Version control/yakubu.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+I am Yakubu Aminat Jumoke.
```
Comment (djubreel) [inline submissions/Dufuna18-master/submissions/Yakubu Aminat Jumoke/Version control/yakubu.txt]: This is the correct file, leave this as it is the valid submission

File: submissions/Version control/yakubu.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+I am Yakubu Aminat Jumoke. 
+I am from kogi state.
```
Comment (djubreel) [inline submissions/Version control/yakubu.txt]: Remove this file

**Pull request closed without merging**